java/query/src/main/java/sleeper/query/lambda/SqsQueryProcessorLambda.java

```diff
@@ -99,8 +102,7 @@ private void updateProperties(String configBucket) throws ObjectFactoryException
             LOGGER.error("Config Bucket was null. Was an environment variable missing?");
             throw new RuntimeException("Error: can't find S3 bucket from environment variable");
         }
-        instanceProperties = loadInstanceProperties(s3Client, configBucket);
-        TablePropertiesProvider tablePropertiesProvider = new TablePropertiesProvider(instanceProperties, s3Client, dynamoClient);
```

Comment:
Is there still a need to reload the instance properties?

Reply:
I've re-added it and tested